cli/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "xstate-cli",
+  "version": "0.0.1",
+  "bin": {
+    "xstate": "./bin/index.js"
+  },
+  "scripts": {
+    "prepare": "esbuild src/bin.ts --outfile=bin/index.js --bundle --platform=node --external:./node_modules/* --external:../node_modules/*"
+  },
+  "devDependencies": {
+    "@types/node": "^16.4.0",
+    "esbuild": "^0.14.21",
+    "xstate": "^4.29.0"
+  },
+  "peerDependencies": {
+    "xstate": "^4.29.0"
+  },
+  "dependencies": {
+    "chokidar": "^3.5.3",
+    "commander": "^8.0.0",
+    "fast-glob": "^3.2.11",
+    "xstate-parser-demo": "0.5.3",
+    "xstate-tools-shared": "*"
```

Comment:
Since we plan to release this we shouldn't use `*` here - it should use a more strict dependency range.